Sprint-1/interpret/to-pounds.js

```diff
@@ -17,11 +17,17 @@ const pence = paddedPenceNumberString
 
 console.log(`£${pounds}.${pence}`);
 
-// This program takes a string representing a price in pence
+// This prograpaddedPenceNumberString takes a string representing a price in pence
 // The program then builds up a string representing the price in pounds
 
 // You need to do a step-by-step breakdown of each line in this program
 // Try and describe the purpose / rationale behind each step
 
 // To begin, we can start with
 // 1. const penceString = "399p": initialises a string variable with the value "399p"
+// 2. Line 3 Removes trailing the p
+// 3. Line 8 added 0 to front of 399 to become 0399
```

Comment:
What does `console.log(paddedPenceNumberString)` shows? What would the value of `paddedPenceNumberString` when `penceString` is "1234p" (or "1p")?

Reply:
> What does `console.log(paddedPenceNumberString)` shows? What would the value of `paddedPenceNumberString` when `penceString` is "1234p" (or "1p")?

Answer 1: The expression in line 8 will append the require number of zeros to any penceString value less than 100 to make it 3 characters, but will not append to penceString values of 3 characters or more.
Answer 2: 1234p => 1234
Answer 3: 1p => 001